docs/Skin/change_suggestions.md

```diff
@@ -0,0 +1,241 @@
+
+**4 - dermis &rarr; eccrine sweat gland**
+
+This looks correct, but both in Uberon and FMA is part of the epidermis. It will need to be edited.
+
+**5 - dermis &rarr; apocrine gland**
+
+This looks correct, but both in Uberon and FMA is part of the epidermis. It will need to be edited.
+
+**6 - dermis &rarr; vasculature of integument**
+
+vasculature of integument is too general. A new term is needed.
+
+**7 - skin of body &rarr; hypodermis**
+
+I have found conflicting information. There are sources [that support this relationship](PMID: 30726010). However, there are others that consider the hypodermis ("under the skin" in greek) as part of the integument, but not part of the skin. Uberon follows the second view. However, I might be able to add a relationship that can validate.
+
+**8 - hypodermis &rarr; skin nerve field**
+
+The author's lable 'nerve' and the Uberon term 'skin nerve field' are not equivalent. The 'skin nerve field' is innerveted by nerves.
+
+
+## CT&rarr;CT
+
+
+
+**No issues**
+
+
+
+## AS&rarr;CT
+
+**1 - skin nerve field &rarr; Neuron**
+
+Neuron is too broad. Is the relationship expected `innervates`? There has been some discussion recently about using relationships that are not 'part of' and they are discouraged.
+
+**2 - vasculature of integument &rarr; endothelial cell**
+
+'endothelial cell' is too broad. A compund term would be the best solution, otherwise a new term is required.
+
+**3 - hypodermis &rarr; B cell**
+
+For immune system no action is planned yet. Compund terms can be a solution.
+
+**4 - stratum granulosum of epidermis &rarr; keratinocyte**
+
+The specific keratinocyte for the stratum granulosum in CL is `granular cell of epidermis` (CL:0002189), it can be used instead of keratinocyte.
+
+**5 - hypodermis &rarr; macrophage**
+
+For immune system no action is planned yet. Compund terms can be a solution.
+
+**6 - hypodermis &rarr; neutrophil**
+
+For immune system no action is planned yet. Compund terms can be a solution.
+
+**7 - hypodermis &rarr; mast cell**
+
+For immune system no action is planned yet. Compund terms can be a solution.
+
+**8 - stratum granulosum of epidermis &rarr; epidermal Langerhans cell**
+
+A 'has part' axiom can be added. Need to confirm that they are present across species in the stratum granulosum.
+
+**9 - stratum spinosum of epidermis &rarr; epidermal Langerhans cell**
+
+A 'has part' axiom can be added.
+
+**10 - stratum spinosum of epidermis &rarr; Merkel cell**
+
+Merkel cells are only found in the stratum basale.
+
+**11 - stratum basale of epidermis &rarr; epidermal Langerhans cell**
+
+A 'has part' axiom can be added. Need to confirm that they are present across species in the stratum basale.
+
+**12 - stratum basale of epidermis &rarr; melanocyte of skin**
+
+That is correct, it should be edited (overlaps [Cichorek et al., 2013](https://pubmed.ncbi.nlm.nih.gov/24278043)) or add 'has part'.
+
+![alt text](image.png)
```

Comment:
can you please move the image to `docs/Skin/assets`?

Reply:
Please, give a more self-explaining file name.

Reply:
Moved and renamed, thank you!